configure.ac

```diff
@@ -63,7 +63,13 @@ AC_ARG_ENABLE([werror],
 AC_ARG_ENABLE([debug],
     [AS_HELP_STRING([--enable-debug],
                     [Turn on debug output])],
-    [debug=$enableval], [debug=no])
+    [debug=yes], [debug=no])
```

Comment:
I think debug=$enableval is fine.  If we do debug=yes here, it breaks --disable-debug.